File: A_Buttons.cpp
#include <bits/stdc++.h>
using namespace std;

typedef long long ll;
#define endl '\n'
#define optimize() ios::sync_with_stdio(false); cin.tie(nullptr);

int main() 
{
    optimize();

    int a, b;
    cin >> a >> b;
    cout << max(a, b) + max(max(a, b) - 1, min(a, b)) << endl;

    return 0;
}

File: A_Buttons.py
# a, b = map(int, input().split())

# ans = max(a, b) + max(max(a, b) - 1, min(a, b))
# print(ans)


a, b = map(int, input().split())
if (a < b):
    a, b = b, a

ans = a + a if a == b else a + a - 1
print(ans)